gitgud/__main__.py

```diff
@@ -293,6 +298,7 @@ def handle_commit(self, args):
         print('Simulating: git commit -m "{}"'.format(commit_name))
 
         commit = self.file_operator.add_and_commit(commit_name)
+        track_commit(self.file_operator, commit_name, commit.hexsha[:7])
```

Comment:
This is currently the only place where we use `add_and_commit` currently, and I think in the future, we'll want to track every commit that we create with the `add_and_commit` function, so put `track_commit` in `add_and_commit`

Reply:
Removed track_commit from handle_commit, moved to add_and_commit.